src/module/canvas/layer/template.ts

```diff
@@ -6,13 +6,26 @@ export class TemplateLayerPF2e<
     /** Preview event listeners that can be referenced across methods */
     #previewListeners: TemplatePreviewEventListeners | null = null;
 
+    template: MeasuredTemplatePF2e | null = null;
```

Comment:
What is the reason for having one of the layer's contained templates as a single property?

Reply:
To allow the measured template itself to say "hey snap according to me".

In retrospect I should either make that communication occur with a method (and make this property private) or see if layer.hover gets me what I want. Don't merge this yet!